cmd/imageproxy/main.go

```diff
@@ -39,6 +39,7 @@ var denyHosts = flag.String("denyHosts", "", "comma separated list of denied rem
 var referrers = flag.String("referrers", "", "comma separated list of allowed referring hosts")
 var includeReferer = flag.Bool("includeReferer", false, "include referer header in remote requests")
 var followRedirects = flag.Bool("followRedirects", true, "follow redirects")
+var maxRedirects = flag.Uint("maxRedirects", 20, "maximum redirection-followings allowed: 0-254 range. 0 is no limit")
```

Comment:
I'm not totally sure what a good default is for this case, I though 20 is ok as it's similar of browser's defaults.

Reply:
20 sounds great to me, although to upstream this I think 0 would make more sense to be backwards compatible 😅 I don't think it's a big deal, you can try with 20 for the upstream PR as well! If they're ok with it, then perfect!